scripts/extensions/pool_update.precheck

```diff
@@ -149,6 +156,33 @@ def execute_precheck(session, control_package, yum_conf_file, update_precheck_fi
             else:
                 raise PrecheckFailure(errmsg)
         else:
+            regex = ERROR_XML_START + '.+' + ERROR_XML_END
+            m = re.search(regex, output, flags=re.DOTALL)
```

Comment:
I struggle to understand why this is called `m`. Maybe it's easier to just do without defining `m`:

```
if re.search(regex, output, flags=re.DOTALL):
```

If you want to keep it, I'd like to see a better name. 

As I understand it, the regexp is an optimisation: the xml parser is only used when the regexp matches. I wonder whether this is safe. For example, does the regexp still match when the `<error>` node is spread over multiple lines? I would suggest to avoid this optimisation and always use the xml parser to find what you are looking for.